distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/NsiliClient.java

```diff
@@ -755,7 +748,7 @@ public String getIorTextFile(String iorURL) throws Exception {
         }
```

Comment:
Outside the scope of this PR, but probably should have a LOGGER.error() for the case that the client cannot connect to the server.